html/core/src/jsMain/kotlin/org/jetbrains/compose/web/attributes/AttrsScope.kt

```diff
@@ -141,7 +141,26 @@ open class AttrsScopeBuilder<TElement : Element>(
     internal val attributesMap = mutableMapOf<String, String>()
     internal val styleScope: StyleScopeBuilder = StyleScopeBuilder()
     internal val propertyUpdates = mutableListOf<Pair<(Element, Any) -> Unit, Any>>()
-    internal var refEffect: (DisposableEffectScope.(TElement) -> DisposableEffectResult)? = null
+    private val refEffects: MutableList<DisposableEffectScope.(TElement) -> DisposableEffectResult> = mutableListOf()
+    internal val refEffect: (DisposableEffectScope.(TElement) -> DisposableEffectResult)?
+        get() = if (refEffects.isEmpty()) {
+            null
+        } else {
+            { element ->
+                val onDisposes = mutableListOf<DisposableEffectResult>()
+                refEffects.forEach {
+                    onDisposes.add(it(element))
+                }
+
+                onDispose {
+                    onDisposes.forEach { disposableEffectResult ->
+                        runCatching {
```

Comment:
is `runCatching` needed here? what is its purpose? why do we silently ignore an exception here?

Reply:
It is required here to avoid memory leaks. Disposing is freeing operation and disposes must not affect on each others